### ISSUE TITLE
Runtime: Reagents

### ISSUE THREAD

**Issue by Lohikar** [Aurorastation/Aurora.3#2682]
```
runtime error:
[16:56:26]Cannot execute null.remove reagent().
[16:56:26]proc name: remove self (/datum/reagent/proc/remove_self)
[16:56:26]  source file: Chemistry-Reagents.dm,37
[16:56:26]  usr: null
[16:56:26]  src: Blood (/datum/reagent/blood)
[16:56:26]  call stack:
[16:56:26]Blood (/datum/reagent/blood): remove self(1)
[16:56:26]Blood (/datum/reagent/blood): on mob life(Isaac Catman (/mob/living/carbon/human), null, 3)
[16:56:26]/datum/reagents/metabolism (/datum/reagents/metabolism): metabolize()
[16:56:26]Isaac Catman (/mob/living/carbon/human): handle chemicals in body()
[16:56:26]Isaac Catman (/mob/living/carbon/human): Life()
[16:56:26]Isaac Catman (/mob/living/carbon/human): Life()
[16:56:26]Isaac Catman (/mob/living/carbon/human): Life()
[16:56:26]Mobs (/datum/controller/subsystem/mobs): fire(1)
[16:56:26]Mobs (/datum/controller/subsystem/mobs): ignite(1)
[16:56:26]Master (/datum/controller/master): RunQueue()
[16:56:26]Master (/datum/controller/master): Loop()
[16:56:26]Master (/datum/controller/master): StartProcessing(0)
```

**Comment by skull132** [comment]
This is oddly conditional. Implemented logging for this in #2755.

**Comment by Geevies** [comment]
Fixed?
